### PR TITLE
[JENKINS-37599] Fix handling of "minimum" in f:repeatable and f:repeatableProperty

### PR DESCRIPTION
**Pull request by ikedam**
See [JENKINS-37599](https://issues.jenkins-ci.org/browse/JENKINS-37599), [JENKINS-44824](https://issues.jenkins-ci.org/browse/JENKINS-44824)

### Proposed changelog entries
```
    - type: bug
      message: Internal: "minimum" should default to 0 in <f:repeatable> and <f:repeatableProperty> especially even when they are nested.
      issue: 37599
```

### Submitter checklist

- [x] JIRA issue is well described
- [x] Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
      * Use the `Internal: ` prefix if the change has no user-visible impact (API, test frameworks, etc.)
- [x] Appropriate autotests or explanation to why this change has no tests
- [x] For dependency updates: links to external changelogs and, if possible, full diffs

### CODE REVIEW

File: test/src/test/java/lib/form/RepeatablePropertyTest.java [hunk 1]
```diff
@@ -69,6 +76,32 @@ public void testFieldNotNullWithDefaultIgnoresDefaults() throws Exception {
...
+        });
+        List<HtmlTextInput> greatPropertyInputs = form.getElementsByAttribute("input", "type", "text");
+        CollectionUtils.filter(greatPropertyInputs, new Predicate<HtmlTextInput>() {
+            @Override
+            public boolean evaluate(HtmlTextInput input) {
+                return input.getNameAttribute().endsWith(".greatProperty");
+            }
+        });
+        assertEquals(1, containerNameInputs.size());
+        assertEquals(0, greatPropertyInputs.size());
```
Comment (ikedam) [inline test/src/test/java/lib/form/RepeatablePropertyTest.java]: Without this fix, this fails with "junit.framework.AssertionFailedError: expected:<0> but was:<1>" as `minimum="1"` in 
RepeatablePropertyTest/nested.jelly is unexpectedly propagated to ExcitingObjectContainer/config.jelly .

**Comment by ikedam** [comment]
"core.ViewTest.renameJob" passed in my environment though it failed on Jenkins.
That test looks unstable and seems sometime fail. It failed also in the master branch: https://ci.jenkins.io/job/Core/job/jenkins/job/master/1029/

**Review by oleg-nenashev — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
Looks great

**Review by daniel-beck — APPROVED** [review]

**Comment by ikedam** [comment]
Thank you!

**Pull request merged**